src/pages/PageNotFound.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+export const PageNotFound: React.FC = () => {
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        left: '50%',
+        top: '50%',
+        transform: 'translate(-50%, -50%)'
+      }}
```

Comment:
Can you attach a screenshot of what this page looks like when running the app locally?

Reply:
![image](https://user-images.githubusercontent.com/55262107/111109624-810df780-8531-11eb-907e-aca1310e2c23.png)

Reply:
Looks great!